website/src/react-demo-runner.js

```diff
@@ -27,6 +27,16 @@ export default class DemoRunner extends Component {
     const {width, height, name, demo, noPanel, sourceLink} = this.props;
     const controls = demo.getInfo && demo.getInfo();
 
+    const notSupported = demo.isNotSupported && demo.isNotSupported();
```

Comment:
Nit: `is*` functions usually are expected to return boolean. Make it two flags `webGL2Only` and `altText`?